lib/cisco_node_utils/cmd_ref/pim_rp_address.yaml

```diff
@@ -0,0 +1,32 @@
+# pim_rp_address
```

Comment:
Let's move all of these entries into `pim.yaml`. In most cases it's easier to have all related tags in one file. In this case you've got an identical _template so there's no technical reason to keep them separate.